account_asset_operating_unit/report/account_asset_report_xls.py

```diff
@@ -28,20 +28,16 @@ def _get_asset_template(self):
     def _get_assets(self, wiz, data):
         super()._get_assets(wiz, data)
         if wiz.operating_unit_id:
-            # Filter asset by operating unit
             assets = data["assets"]
             assets_by_ou = assets.filtered(
                 lambda a: not a.operating_unit_id
                 or a.operating_unit_id == wiz.operating_unit_id
```

Comment:
I think if filter by operating unit should show only assets that match with selected opearting unit on the wizard. Should not show assets that no have operating unit.

this line, if I change to
```
assets_by_ou = assets.filtered(lambda a: a.operating_unit_id == wiz.operating_unit_id)
```

Is this ok for you ? @ps-tubtim

Reply:
@newtratip I agree with you. I already change it.